### PR TITLE
[INTERNAL] Schema: Update error messages / fix framework schema

### PR DESCRIPTION
**Pull request by matz3**
**Thank you for your contribution!** 🙌

To get it merged faster, kindly review the checklist below:

## Pull Request Checklist
- [ ] Reviewed the [Contributing Guidelines](https://github.com/SAP/ui5-tooling/blob/master/CONTRIBUTING.md#-contributing-code)
    + Especially the [How to Contribute](https://github.com/SAP/ui5-tooling/blob/master/CONTRIBUTING.md#how-to-contribute) section 
- [ ] [No merge commits](https://github.com/SAP/ui5-tooling/blob/master/docs/Guidelines.md#no-merge-commits)
- [ ] [Correct commit message style](https://github.com/SAP/ui5-tooling/blob/master/docs/Guidelines.md#commit-message-style)

### CODE REVIEW

File: test/lib/validation/schema/specVersion/2.0/kind/project/application.js [hunk 1]
```diff
@@ -630,3 +630,24 @@ test("framework configuration: Invalid", async (t) => {
...
+		"type": "application",
+		"metadata": {
+			"name": "my-application"
+		},
+		"framework": {}
+	}, [
+		{
+			dataPath: "/framework",
+			keyword: "required",
+			message: "should have required property 'name'",
```
Comment (KlattG) [inline test/lib/validation/schema/specVersion/2.0/kind/project/application.js]: must have required property 'name'

File: test/lib/validation/schema/specVersion/2.0/kind/project/library.js [hunk 2]
```diff
@@ -702,3 +702,24 @@ test("framework configuration: Invalid", async (t) => {
...
+		"type": "library",
+		"metadata": {
+			"name": "my-library"
+		},
+		"framework": {}
+	}, [
+		{
+			dataPath: "/framework",
+			keyword: "required",
+			message: "should have required property 'name'",
```
Comment (KlattG) [inline test/lib/validation/schema/specVersion/2.0/kind/project/library.js]: must have required property 'name'

File: test/lib/validation/schema/specVersion/2.0/kind/project/application.js (hunk 1 above)
Comment (matz3) [reply test/lib/validation/schema/specVersion/2.0/kind/project/application.js]: That's the message coming from the "ajv" library. It's just here as we compare the whole error objects returned by ajv. But we won't use it to create our messages.

File: test/lib/validation/schema/specVersion/2.0/kind/project/library.js (hunk 2 above)
Comment (matz3) [reply test/lib/validation/schema/specVersion/2.0/kind/project/library.js]: Same here. Sorry for the confusion.

**Pull request merged**